test/IECoreHoudini/CortexConverterSop.py

```diff
@@ -44,6 +44,9 @@
 
 class TestCortexConverterSop( IECoreHoudini.TestCase ):
 
+	ToCortex = 0
```

Comment:
There is an enum in the SOP but there isn't a binding for this right?

Reply:
Lets add a binding ;) Or can we resuse `IECoreHoudini.SceneCacheNode.GeometryType`? Maybe that's not suitable...

Reply:
> Or can we resuse IECoreHoudini.SceneCacheNode.GeometryType? Maybe that's not suitable...

I think using GeometryType is worse as it has a couple of values which are invalid. I was just improving the readability of the tests rather than changing any API so I don't think it's worth it. Unless we import this binding in houdini for working with the SOP node?

Reply:
Ok, lets ignore it